mono/mini/liveness.c

```diff
@@ -37,6 +37,62 @@
 static void mono_analyze_liveness2 (MonoCompile *cfg);
 #endif
 
+
+#define INLINE_SIZE 16
```

Comment:
did you play around with other values?

Reply:
Yes, I measured 8 and 32 with Roslyn and 32 was the best one.

Reply:
why 16 then? 😄 or did you mean 16 instead of 32?

Reply:
What I meant was that I tried 8, 16 and 32. The winner was 16.